subprojects/docs/src/docs/userguide/dep-man/03-controlling-transitive-dependencies/platforms.adoc

```diff
@@ -114,6 +114,15 @@ include::sample[dir="snippets/dependencyManagement/catalogs-settings/groovy",fil
 include::sample[dir="snippets/dependencyManagement/catalogs-settings/kotlin",files="build.gradle.kts[tags=use_version]"]
 ====
 
+If the alias of a declared version is also a prefix of some more specific alias, as in `version("zinc", "...")"` and `version("zinc-apiinfo", "...")`, then
+the value of the more generic version is available via link:{javadocPath}/org/gradle/api/provider/ProviderConvertible.html#asProvider--[`asProvider()`] on the type-safe accessor:
+
+.Using a version from a version catalog when there are more specific versions
```

Comment:
I would use "aliases" instead of versions as the last word. This would be more consistent with the paragraph above.